include/controller.h

```diff
@@ -148,7 +143,7 @@ class controller {
 
 	ConfigPaths configpaths;
 
-	Reloader reloader;
+	std::unique_ptr<Reloader> reloader;
```

Comment:
Sorry, I don't understand why you could possibly make that change. Can you explain?

Reply:
Hmm. The other way to do this I guess is define a `operator=` for Reloader and then re-initialize it after `rsscache` creation. Is that what you had in mind? Am I missing something?

Reply:
Oh, sorry, I assumed `cache` is a non-pointer field of `controller`, and thought you could initialize `Reloader` in `controller`'s member initialization list. My bad.